libero/src/main/groovy/com/connexta/libero/Config.groovy

```diff
@@ -3,28 +3,35 @@ package com.connexta.libero
 import static org.ho.yaml.Yaml.load
 
 class Config {
-    Map projectProperties = new HashMap()
+
     String projectDir
     String projectName
-    File configFile
     String sourceRemote
     String destRemote
     String ref
     String destBranch
+
     String startVersion
     String releaseVersion
     String nextVersion
+    String releaseName
+
     String mavenRepo
     Map preProps
     Map postProps
-    String releaseName
+
     boolean quickBuild
     boolean gitPush
     boolean dryRun
     boolean force
+
     Util util = new Util()
 
-    public loadConfig = { ->
+    Map projectProperties = new HashMap()
+    File configFile
```

Comment:
We only keep `configFile` around so we can print it in `printConfig`. Do you think it's necessary? If we decide to keep it, we should at least make it `private` so no one can change it from the outside.